clang/test/SemaSYCL/spec_const_accesors.cpp

```diff
@@ -0,0 +1,46 @@
+// RUN: %clangxx -fsycl-device-only %s -o %t.bc
+//
+//==----------- spec_const_accessors.cpp -----------------------------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
```

Comment:
AFAIK adding this header to tests is not necessary.
```suggestion
```

Reply:
Done